### DISCUSSION TITLE
socket.io in android client

### DISCUSSION THREAD

**Discussion by majidkazemi2000** [socketio/socket.io#3870] [Q&A]
why socket.io client can not connect to server in api 28+ in android ? (client version 2 & server version 4)

**Comment by darrachequesne** [comment]
That sounds weird. Do you have any logs? Any errors thrown?

**Reply by majidkazemi2000** [reply]
no , there is no error ,
 private Socket mSocket;{
        try{
            mSocket = IO.socket("http://myip:3000");
        } catch (URISyntaxException e) {
            Log.i("error","error in here");
        }
    }


msocket.connect();



this is my code and in api 28+ not connection with server but in 27- connection is ok !!
